### DISCUSSION TITLE
Tooltips not showing when using Bootstrap

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#36558] [Q&A] [answered]
New to bootstrap...

I have linked up the CSS and JS Bootstrap files... and copied the boilerplate template code (see below)

```
<button type="button" class="btn btn-secondary" data-bs-toggle="tooltip" data-bs-placement="top" title="Tooltip on top">
  Tooltip on top
</button>
```

Yet, somehow my tooltips aren't showing. Have been trying to figure it out for more than 1 hour, appreciate any inputs on this.

**Comment by melvincwng** [comment]
@waikitmun, what version of Bootstrap are you using?

**Reply by ghost** [reply]
v5.1

**Comment by melvincwng** [comment, marked as answer]
@waikitmun, you might be missing this piece of code (without this, the tooltips won't be enabled and hence won't show):

```
var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
  return new bootstrap.Tooltip(tooltipTriggerEl)
})
```

Reference: https://getbootstrap.com/docs/5.1/components/tooltips/ ('Example: Enable Tooltips anywhere')

**Reply by ghost** [reply]
Will try it, thanks!
